.github/workflows/upstream.yml

```diff
@@ -55,8 +55,72 @@ jobs:
         run: source continuous_integration/scripts/install.sh
 
       - name: Run tests
+        id: run_tests
         shell: bash -l {0}
         run: source continuous_integration/scripts/run_tests.sh
 
       - name: Coverage
         uses: codecov/codecov-action@v1
+
+  report:
+    name: report
+    needs: build
+    if: |
+      always()
+      && github.event_name == 'schedule'
```

Comment:
For example, I _think_ we could temporarily comment out this `github.event_name == 'schedule'` condition and then push an empty commit with `test-upstream` in the commit message to trigger an upstream build

Reply:
Also, it might be good to try this out before @jsignell fixes all the upstream tests : )  (xref https://github.com/dask/dask/pull/8037)

Reply:
lol that'll be a while...